### ISSUE TITLE
Docker Hub is not being updated

### ISSUE THREAD

**Issue by dbpolito** [ochinchina/supervisord#234]
Docker Image has version `v0.5`, probably the integration for sync / build is broken.

**Comment by sgaunet** [comment]
Yes, please, update the image on docker hub, that's a very nice way to get the binary.

**Comment by shyim** [comment]
Had the same problem cause I needed a arm64 image. Have created a PR to update the current Dockerfile
https://github.com/ochinchina/supervisord/pull/235
